src/GeneratedMapper/MapperGenerator.cs

```diff
@@ -39,23 +42,55 @@ public void Execute(GeneratorExecutionContext context)
                 // resolve all mappings that need other mappings information (like nested mappings)
                 ResolvePendingNestedMappings(foundMappings);
 
-                ValidateMappings(context, foundMappings);
+                var distinctMappings = foundMappings
+                    .GroupBy(x => x, MappingInformation.SourceTypeDestinationTypeComparer)
+                    .Select(x => x.OrderByDescending(m => m.AttributeIndex).First()).ToArray();
```

Comment:
Why have you chosen for picking the highest index? Don't you feel like that `Index = 1` would be the best match?

Reply:
Extensions index to `Int32.MinValue`, so I didn't want that to be the first, but yea it could be max and then could `OrderBy` making first be 1.  I just thought negative index's for extensions would differentiate them from attribute index's and made code work off that assumption.

I'll probably change it so index 1 is first and make index max for `MapTo<,>` calls.

Reply:
Fixed already